common/culture/name_lists/wc_darnassian.txt

```diff
@@ -77,7 +77,7 @@
 	male_names = {
 		Aelgorn Aldryth Allendril Altruis Andalar Andoril Anelas Anren Arantir Ardan Aren Arvell Ashelan Astalor Astran Athridas Auran Baelir Balthule  
 		Baritanas Broll Cerellean Corithras Delandros Deldren Dralith Elerion Elloric Eranril Erelas Erion Eriun Evenar Faeolas Falorn   
-		Falrin Farodin Feronas Ferryn Filandras Firondren Foriel Fyldan Gaerolas Galind Gilshalan Golras Halsteth Idras Isoraen Ithalaine Ithis    
```

Comment:
Ithis is a night elf name
https://warcraft.wiki.gg/wiki/Ithis_Moonwarden

Reply:
ok will fix this

Reply:
fixed